cpp/harvest/marc_issn_lookup.cc

```diff
@@ -50,6 +50,47 @@ struct DebugInfo {
     DebugInfo() = default;
 };
 
+struct TitleInfo773 {
+    std::string title_, subfield_w_;
+    char ind1_, ind2_;
```

Comment:
indicators should not be stored in the cache but checked during iteration. only 0 8 is relevant.